addons/l10n_cl/views/report_invoice.xml

```diff
@@ -1,65 +1,247 @@
+                                    <h6 t-att-style="'color: %s;' % o.company_id.primary_color">
+                                        <strong t-att-style="'color: %s;' % o.company_id.primary_color">
+                                            <br/>
+                                            <span style="line-height: 180%;">RUT:</span>
+                                            <span t-field="o.company_id.partner_id.vat"/>
+                                            <br/>
+                                            <span class="text-uppercase" t-esc="report_name"/>
+                                            <br/>
+                                            <span>Nº:</span>
+                                            <span style="line-height: 200%;" t-esc="report_number"/>
+                                        </strong>
+                                    </h6>
+                                </div>
+                            </div>
+                            <!-- (6) Titulo de Documento -->
+                            <div class="row text-center">
+                                <div class="col-12 text-center" t-att-style="'color: %s;' % o.company_id.primary_color"
+                                     name="regional-office"/>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
+
+            </div>
+
+        </div>
+    </template>
+
+
+    <template id="informations">
+        <div id="informations" class="row mt8 mb8">
+            <div class="col-6">
+                <strong>
+                    <span t-att-style="'color: %s;' % o.company_id.secondary_color">Date:</span>
+                </strong>
+                <span t-esc="report_date" t-options='{"widget": "date"}'/>
+                <br/>
+
+                <strong>Customer:</strong>
+                <span t-field="o.partner_id.name"/>
+                <br/>
+
+                <t t-if="o.partner_id.vat and o.partner_id.l10n_latam_identification_type_id">
+                    <strong>
+                        <t t-esc="o.partner_id.l10n_latam_identification_type_id.name or o.company_id.country_id.vat_label" id="inv_tax_id_label"/>:
+                    </strong>
+                    <span t-esc="o.partner_id.vat"/>
+                    <br/>
+                </t>
+                <strong>GIRO:</strong>
+                <span t-esc="o.partner_id.industry_id.name or ''"/>
+                <br/>
+            </div>
+            <div class="col-6">
+                <strong>Due Date:</strong>
+                <span t-esc="o.invoice_date_due" t-options='{"widget": "date"}'/>
+                <br/>
+                <strong>Address:</strong>
+                <span t-field="o.partner_id"
+                      t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;], &quot;no_marker&quot;: true, &quot;no_tag_br&quot;: True}"/>
```

Comment:
No need for &quot;  We can use regular ' or "  (should have been adapted for l10n_ar as well)

Reply:
TODO

Reply:
solved